src/mocks/handlers.tsx

```diff
@@ -146,4 +146,97 @@ export const handlers = [
       })
     )
   ),
+  graphql.query('Tags', (req, res, ctx) =>
+    res(
+      ctx.data({
+        user: {
+          userSettings: {
```

Comment:
Check the list of tags returned from the query https://www.wallora.com/api/tags?searchstring=.* to get an idea for the mock data structure.
This API call returns a list of tags specific to that user and can be filtered with a searchstring